docs/pktoken.md

```diff
@@ -0,0 +1,702 @@
+# A Guide to PK Tokens
+
+OpenPubkey works by binding an public key to an identity described in an ID Token. To provide verifiers with all the needed information to verify this binding between identity and public key we include additional information by extending the ID Token with additional signatures. This is possible because ID Tokens are JSON Web Signatures (JWS) and JWS can support more than one signature. We call an ID Token extended in this fashion a PK Token (Public Key Token).
```

Comment:
```suggestion
OpenPubkey works by binding a public key to an identity described in an ID Token. To provide verifiers with all the needed information to verify this binding between identity and public key we include additional information by extending the ID Token with additional signatures. This is possible because ID Tokens are JSON Web Signatures (JWS) and can support multiple signatures. We call an ID Token extended in this fashion a PK Token (Public Key Token).
```